app/assets/javascripts/utils/course_date_utils.js

```diff
@@ -21,59 +20,53 @@ const CourseDateUtils = {
       timeFormat = ' HH:mm';
       try {
         timeZoneAbbr = ' ';
-        timeZoneAbbr += moment(datetime).toDate().toString().split('(')[1].slice(0, -1);
+        timeZoneAbbr += datetime.toString().split('(')[1].slice(0, -1);
       } catch (err) {
         timeZoneAbbr = '';
       }
     }
-    const format = `YYYY-MM-DD${timeFormat}`;
-    return moment(datetime).format(format) + timeZoneAbbr;
+    const dateFormat = `yyyy-MM-dd${timeFormat}`;
+    return format(datetime, dateFormat) + timeZoneAbbr;
   },
 
   // Returns an object of minDate and maxDate props for each date field of a course
   dateProps(course) {
-    const startDate = moment(course.start, 'YYYY-MM-DD');
-
+    const startDate = parseISO(course.start);
     const props = {
       end: {
         minDate: startDate
       },
       timeline_start: {
         minDate: startDate,
-        maxDate: moment(course.timeline_end, 'YYYY-MM-DD')
+        maxDate: parseISO(course.timeline_end)
       },
       timeline_end: {
-        minDate: moment(course.timeline_start, 'YYYY-MM-DD'),
-        maxDate: moment(course.end, 'YYYY-MM-DD')
+        minDate: parseISO(course.timeline_start),
+        maxDate: parseISO(course.end)
       }
     };
-
     return props;
   },
 
   // This method takes a current version of a course and an updated key-value pair
   // for changing one of the date fields and returns a course where all the dates
   // are consistent with each other.
   updateCourseDates(prevCourse, valueKey, value) {
-    const updatedCourse = $.extend({}, prevCourse);
+    const updatedCourse = Object.assign({}, prevCourse);
     updatedCourse[valueKey] = value;
     // Just return with the new value if it doesn't pass validation
     // or if it it lacks timeline dates
     if (!this.isDateValid(value) || !updatedCourse.timeline_start) { return updatedCourse; }
-
-    if (moment(updatedCourse.start, 'YYYY-MM-DD').isAfter(updatedCourse.timeline_start, 'YYYY-MM-DD') && valueKey !== 'timeline_start') {
+    if (isAfter(toDate(updatedCourse.start), toDate(updatedCourse.timeline_start)) && valueKey !== 'timeline_start') {
       updatedCourse.timeline_start = updatedCourse.start;
     }
-    if (moment(updatedCourse.timeline_start, 'YYYY-MM-DD').isAfter(updatedCourse.timeline_end, 'YYYY-MM-DD') && valueKey !== 'timeline_end') {
+    if (isAfter(toDate(updatedCourse.timeline_start), toDate(updatedCourse.timeline_end)) && valueKey !== 'timeline_end') {
       updatedCourse.timeline_end = updatedCourse.timeline_start;
     }
-    if (moment(updatedCourse.timeline_end, 'YYYY-MM-DD').isAfter(updatedCourse.end, 'YYYY-MM-DD') && valueKey !== 'end') {
+    if (isAfter(toDate(updatedCourse.timeline_end), toDate(updatedCourse.end)) && valueKey !== 'end') {
       updatedCourse.end = updatedCourse.timeline_end;
     }
-    if (moment(updatedCourse.start, 'YYYY-MM-DD').isAfter(updatedCourse.timeline_start, 'YYYY-MM-DD') && valueKey !== 'timeline_start') {
```

Comment:
All the other parts of this are being converted to not use `moment` but this one is just being removed. Does that change the behavior?

Reply:
Yeah, it looks like that's the case. I see I added it in https://github.com/WikiEducationFoundation/WikiEduDashboard/commit/2b9ae07a8764eba7c84ec7d44da78c7a2d46e86e but I guess that duplicated part must have been useless. I can't think of any way this would change behavior.